### PR TITLE
BUGFIX: Do not use the removed DbalClientInterface

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

File: Classes/Infrastructure/Healthcheck/CrHealthcheck.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
 class CrHealthcheck implements HealthcheckInterface
 {
     public function __construct(
-        private DbalClientInterface $dbalClient,
+        private Connection           $dbalConnection,
```
Comment (bwaidelich) [inline Classes/Infrastructure/Healthcheck/CrHealthcheck.php]: Just a tiny cosmetic fix
```suggestion
        private Connection $dbalConnection,
```

**Review by bwaidelich — APPROVED** [review]
Thanks!

**Comment by mhsdesign** [comment]
btw this should go in 4 :O

**Comment by mhsdesign** [comment]
but lol yolo idk

**Pull request merged**